src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletService.java

```diff
@@ -1,4 +1,9 @@
 package pl.byczazagroda.trackexpensesappbackend.service;
 
+import pl.byczazagroda.trackexpensesappbackend.dto.UpdateWalletDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.WalletDTO;
+
 public interface WalletService {
+
+    WalletDTO updateWallet(UpdateWalletDTO walletToEdit);
```

Comment:
We can change name of the argument to `walletToUpdate` as we do in `WalletServiceImpl.java`

Reply:
Updated.